azurerm/internal/services/servicefabric/resource_arm_service_fabric_cluster.go

```diff
@@ -220,6 +220,29 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				},
 			},
 
+			"client_certificate_common_name": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"certificate_common_name": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+						},
+						"certificate_issuer_thumbprint": {
```

Comment:
I think we can drop the certificate
```suggestion
						"issuer_thumbprint": {
```